Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -2865,11 +2865,9 @@ AccessibilityRole AccessibilityNodeObject::determineAriaRoleAttribute() const
     if (role == AccessibilityRole::Presentational && supportsARIAAttributes())
         role = AccessibilityRole::Unknown;
     
-    // The ARIA spec states, "Authors must give each element with role region a brief label that
-    // describes the purpose of the content in the region." The Core AAM states, "Special case:
-    // if the region does not have an accessible name, do not expose the element as a landmark.
-    // Use the native host language role of the element instead."
-    if (role == AccessibilityRole::LandmarkRegion && !hasAttribute(aria_labelAttr) && !hasAttribute(aria_labelledbyAttr))
+    // The ARIA spec states, "In situation when an author has not specified names for these landmarks such as form and region, it is considered an authoring error.
+    // If a valid fallback role had been specified, or if the element had an implicit ARIA role, then user agents would continue to expose that role, instread."
```

Comment:
Might be nice to link to the spec here. https://w3c.github.io/aria/#document-handling_author-errors_roles

I also think "situation" is missing an "s", and there's a typo of "instread" vs. "instead". What do you think about using this as the comment? May need to adjust the line breaks to get it formatted more nicely 🙂 

```
// https://w3c.github.io/aria/#document-handling_author-errors_roles
// In situations where an author has not specified names for the form and
// region landmarks, it is considered an // authoring error. The user agent
// MUST treat such elements as if no role had been provided.
```

Reply:
It has become cleaner.
Thank you for the review.